src/test/mockJwt.ts

```diff
@@ -0,0 +1,25 @@
+import createJWKSMock from 'mock-jwks';
+import { issuer } from '../auth/jwtOptions';
+import type { JWKSMock } from 'mock-jwks';
+
+export const getMockJwks = (): JWKSMock => createJWKSMock(
+  issuer,
+  '/protocol/openid-connect/certs',
+);
+
+export const getMockJwt = (jwksMock: JWKSMock): string => {
+  const aMomentAgo = Math.round(new Date().getTime() / 1000);
+  return jwksMock.token({
+    exp: aMomentAgo + 100000,
```

Comment:
More re-use of the mock JWKS server and mock JWT resolves the issue at test time. Given the re-use, it makes sense to increase the expiration duration.